media-libs/openpgl/openpgl-0.5.0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Intel Open Path Guiding Library"
+HOMEPAGE="https://github.com/OpenPathGuidingLibrary/openpgl"
+SRC_URI="https://github.com/OpenPathGuidingLibrary/openpgl/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+X86_CPU_FLAGS=( sse4_2:sse4_2 avx2:avx2 avx512dq:avx512dq )
```

Comment:
Am I missing something or are both LHS and RHS parts always the same here? Also, you don't seem to be using RHS anywhere.

Reply:
Ah, I copy pasted this from the embree ebuild without thinking too much about it.
You are correct.